File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1273.cpp
#include <iostream>

using namespace std;

int main() {
    int v, k=0;
    while(true){
        cin >> v;
        if(v==0){
            break;
        }
        
        if(k==0){
            k++;
        } else{
            cout << endl;
        }
        string pal[v];
        int maior = 0;
        for(int i=0;i<v;i++){
            cin >> pal[i];
            if(pal[i].size()>maior){
                maior=pal[i].size();
            }
        }
        for(int i=0;i<v;i++){
            for(int j=0;j<maior-pal[i].size();j++){
                cout << " ";
            }
            cout << pal[i] << endl;
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1273.py
c = 1

while True:
    try:
        n = int(input())
        
        if n == 0:
            break
        elif c != 1:
            print()
            
        maior = 0
        pal = []
        for i in range(n):
            pal.append(input())
            if len(pal[-1]) > maior:
                maior = len(pal[-1])
        
        for i in pal:
            print(i.rjust(maior))
            
        c += 1
        
    except:
        break